src/csvwidget/widget.ts

```diff
@@ -17,16 +17,33 @@ import {
   ABCWidgetFactory, IDocumentModel, IDocumentContext
 } from '../docregistry';
 
+import * as d3Dsv from 'd3-dsv';
+
 import {
-  csvParse
-} from 'd3-dsv';
+  PanelLayout
+} from 'phosphor/lib/ui/panel';
 
 
 /**
  * The class name added to a csv widget.
  */
 const CSV_CLASS = 'jp-CSVWidget';
 
+/**
+ * The class name added to a csv toolbar widget.
+ */
+const CSV_TOOLBAR_CLASS = 'jp-CSVToolbarWidget';
```

Comment:
Given our CSS naming conventions, this would be named `jp-CSVWidget-toolbar`. The convention has the form: `jp-NamedTypeScriptClass-subcomponent`.